idxmgmt/ilm/config.go

```diff
@@ -94,9 +97,13 @@ func NewConfig(info beat.Info, cfg *libcommon.Config) (Config, error) {
 		}
 		m.Index = idx
 		config.Setup.Mappings[et] = m
-	}
-	if len(config.Setup.Policies) == 0 {
-		config.Setup.Policies = defaultPolicies()
+		// if require_policy=false and policy does not exist, add it with an empty body
```

Comment:
Nit: can you please move this directly before the `config.Setup.Policies[m.PolicyName] = ...` line? It's a bit confusing here since the next line is the opposite of what's stated here (`require_policy==true`), which had me wondering if there was a typo.